downstream/modules/eda/proc-eda-set-up-new-decision-environment.adoc

```diff
@@ -0,0 +1,35 @@
+[id="proc-eda-set-up-new-decision-environment"]
+
+= Setting up a new decision environment
+
+Decision environments are a container image to run Ansible rulebooks.
+They create a common language for communicating automation dependencies, and provide a standard way to build and distribute the automation environment.
+The default decision environment is found at link:https://quay.io/repository/ansible/ansible-rulebook[Ansible-Rulebook]. 
+
```

Comment:
suggest changing to "is found in the followed by the link". This will render as "...is found in the Ansible-Rulebook" and will read better.